web/react-web/src/Content/Graph/Graph.js

```diff
@@ -142,9 +149,32 @@ class Graph extends Component {
                 name = `Active cases for ${one.long_name}`;
             }
 
+            let yRatio = 1.0;
+            switch (dataScaling) {
+                case SCALING_SAME_PEAK:
+                    let maxValue = one.max_value;
+                    if (one.type !== 'prediction') {
+                        maxValue = Math.max(...one.values);
+                    }
+                    yRatio = 1.0 / maxValue * 100.0;
+
+                    finalOptions.yAxis.title.text = 'Active cases %';
+                    finalOptions.yAxis.labels.format = '{value}%';
+                    break;
+                case SCALING_PER_CAPITA:
+                    yRatio = 1.0 / (one.population / 100000.0);
```

Comment:
From a usability perspective, 100000 is perhaps not the best. People know the population in millions, so it might be best to show cases per million (as wroldometers.info is doing https://www.worldometers.info/coronavirus/#countries). That way you can easily multiply it back in your head if needed.

Also this has the nice sideffect of 10-times larger numbers, so we get more precision. This also relates to my other comment regarding precision.

Reply:
Done.